packages/create-app/src/utils/template/npm.ts

```diff
@@ -54,8 +32,9 @@ export async function updateCLIDependencies({
     const dependencyOverrides = {
       '@shopify/cli': cliPath,
       '@shopify/app': appPath,
-      '@shopify/cli-kit': cliKitPath,
-      '@shopify/shopify-cli-extensions': extensionsCliPath,
+      '@shopify/cli-kit': await packagePath('cli-kit'),
+      '@shopify/shopify-cli-extensions': await packagePath('ui-extensions-cli'),
+      '@shopify/plugin-ngrok': await packagePath('plugin-ngrok'),
```

Comment:
What about creating constants above for all of them to keep consistency?

Reply:
I debated internally about this. If you felt strongly enough to comment, I'll change it.